.travis.yml

```diff
@@ -122,20 +122,8 @@ matrix:
         - rustup component add rust-src
         - cargo install cargo-xbuild || true
       script:
-        - cargo build --target=x86_64-sun-solaris
-        - cargo build --target=x86_64-unknown-cloudabi
-        - cargo build --target=x86_64-unknown-freebsd
-        - cargo build --target=x86_64-fuchsia
-        - cargo build --target=x86_64-unknown-netbsd
-        - cargo build --target=x86_64-unknown-redox
-        - cargo build --target=x86_64-fortanix-unknown-sgx
-        - cargo xbuild --target=x86_64-unknown-uefi
-        - cargo xbuild --target=x86_64-unknown-hermit
-        - cargo xbuild --target=x86_64-unknown-l4re-uclibc
-        - cargo xbuild --target=x86_64-uwp-windows-gnu
-        - cargo xbuild --target=x86_64-wrs-vxworks
-        # also test minimum dependency versions are usable
-        - cargo generate-lockfile -Z minimal-versions
+        # We test that getrandom and cpurand build for all x86 targets
+        - cd custom/cpurand
```

Comment:
I don't think there are any issues combining `--all` and `--target` options — so long as we're not using `--feature` I think it makes sense to test sub-crates via `--all`.

Reply:
You're right, but unfortunately we can't use `--all` with this crate. Our custom RNGs only compile on certain arches.

However, we can use `--package` to avoid changing the working directory. I did this where it makes sense.